### ISSUE TITLE
Add some kind of callback to modify popup window generation

### ISSUE THREAD

**Issue by hybridherbst** [Looking-Glass/looking-glass-webxr#20]
Right now, the popup is only created, but not stored in any symbol as far as I can tell and so not possible to access from outside, e.g. to attach key listeners, drop handlers, and other nice web things.

Would be great if there was a way to provide a callback for popup creation to modify what that does and allow attaching other logic.

**Comment by BryanChrisBrown** [comment]
agreed on this, we can definitely expose that as a variable similar to the config setup.
